docs/src/faq.md

```diff
@@ -389,22 +406,59 @@ $ kubectl exec -ti pg-italy-1 -c postgres -- psql -x -c "SHOW timezone"
 TimeZone | Europe/Rome
 ```
 
-<!--
 **What is the recommended architecture for best business continuity
 outcomes?**
 
-TODO
+As covered in the ["Architecture"](architecture.md) section, the main
+recommendation is to adopt shared nothing archictures as much as possible, by
+leveraging the native capabilities and resource that Kubernetes provides in a
+single cluster, namely:
+
+- availability zones: spread your instances across different availability zones
+  in the same Kubernetes cluster
+- worker nodes: as a consequence, make sure that your Postgres instances reside
+  on different Kubernetes worker nodes
+- storage: use dedicated storage for each worker node running Postgres
+
+Use at least one standby, preferably at least two, so that you can configure
+synchronous replication in the cluster, introducing RPO=0 for high
+availability.
+
+If you do not have availability zones - normally the case of on-premise
+installations - separate on worker nodes and storage.
+
+Properly setup continuous backup on a local/regional object store.
+
+The same architecture that is in a single Kubernetes cluster can be replicated
+in another Kubernetes cluster (normally in another geographical area or region)
+through the [replica cluster](replica_cluster.md) feature, providing disaster
+recovery and high availability at global scale.
+
+You can use the WAL archive in the primary object store to feed the replica in
+the other region, without having to provide a streaming connection, if the default
+maximum RPO of 5 minutes is enough for you.
+
+**How can instances be stopped or started?**
+
+Please look at ["Fencing"](fencing.md) or ["Hibernation"](declarative_hibernation.md).
+
 
 **What are the global objects such as roles and databases that are
 automatically created by CloudNativePG?**
 
-TODO
+The operator automatically creates a user for the application (by default
+called `app`) and a database for the application (by default called `app`)
+which is owned by the aforementioned user.
 
-Q: Support for management of Postgres roles via declarative
-configuration
+This way, the database is ready for a microservice adoption, as developers
+can control migrations using the `app` user, without requiring *superuser*
+access.
 
-TODO
+Teams can then create another user for read-write operations throught the
+["Declarative role management"](declarative_role_management.md) feature
+and assign the required `GRANT` to the tables.
 
+<!--
```

Comment:
Do we want to talk also about other things like the pgbouncer user management, the automated extensions creation? at least pointing to the docs?